src/species/clicker.spec.js

```diff
@@ -0,0 +1,103 @@
+import Chance from 'chance';
+
+import clicker from './clicker';
+
+jest.useFakeTimers();
+jest.mock('chance', () => {
```

Comment:
Do we need a mock for chance as it's given as a parameters to `species` ?
Why not just do :
```js
const chance =  {
    natural: ({max}) => max,
    pick: types => types[0],
}
```
and then : `const species = clicker({ log: true })(console, chance);`